sys/net/gnrc/sock/ip/gnrc_sock_ip.c

```diff
@@ -41,17 +41,17 @@ int sock_ip_create(sock_ip_t *sock, const sock_ip_ep_t *local,
         if (gnrc_af_not_supported(local->family)) {
             return -EAFNOSUPPORT;
         }
-        memcpy(&sock->local, local, sizeof(sock_ip_t));
+        memcpy(&sock->local, local, sizeof(sock_ip_ep_t));
```

Comment:
Take a look at all the memset's and memcpy's. We could use some consistency.

Yes, of course you must know what the variable is. But hey, we're programmers. We know. :-)